find_dimensions_of_matrix.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+from numpy import array
+
+
+def find_dimensions_of_matrix(matrix: array) -> str:
+    """
+    The function finds the number of rows and cols of a given matrix
+    :param matrix: A given matrix.
+    :return: A string indicates the rows and cols of the given matrix.
+    """
+    return f"The number of rows and cols of the given matrix are: {matrix.shape}"
```

Comment:
What is the use of returning a string? Wouldn't it make more sense to return just the shape so that it is reusable?